web_access_rule_buttons/models.py

```diff
@@ -19,6 +19,12 @@ def check_access_rule_all(self, operations=None):
         operations = ['read', 'create', 'write', 'unlink']
     result = {}
     for operation in operations:
+        if self.is_transient() and not self.ids:
```

Comment:
They do. But the wizard's form get created before they have one. This check is necessary.